vpn/vpn.proto

```diff
@@ -0,0 +1,197 @@
+syntax = "proto3";
```

Comment:
They do, but I don't like it.  There is too little going on in the package, and golang convention is to import things as the last name component of the package path, so we're constantly having to rename them to `agentproto` and `tailnetproto` to avoid name conflicts in individual files.